pkg/controller/infinispan/xsite.go

```diff
@@ -136,6 +145,18 @@ func appendKubernetesRemoteLocation(infinispan *ispnv1.Infinispan, remoteLocatio
 		"host", host,
 		"port", port,
 	)
+	remoteLocation.Host = host
+	remoteLocation.Port = port
+	return appendBackupSite(logger, remoteLocation, xsite)
+}
+
+func appendBackupSite(logger logr.Logger, remoteLocation *ispnv1.InfinispanSiteLocationSpec, xsite *config.XSite) error {
```

Comment:
logger logr.Logger is redundant here and not using
```suggestion
func appendBackupSite(remoteLocation *ispnv1.InfinispanSiteLocationSpec, xsite *config.XSite) error {
```

Reply:
> logger logr.Logger is redundant here and not using

Ah forgot to remove that 👍 I was considering adding a log statement in this function